src/main/java/org/gridsuite/securityanalysis/server/SecurityAnalysisController.java

```diff
@@ -118,4 +118,20 @@ private static SecurityAnalysisParameters getNonNullParameters(SecurityAnalysisP
         Mono<Void> result = service.deleteResults();
         return ResponseEntity.ok().body(result);
     }
+
+    @GetMapping(value = "/results/{resultUuid}/status", produces = APPLICATION_JSON_VALUE)
+    @Operation(summary = "Get the security analysis status from the database")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The security analysis status")})
+    public ResponseEntity<Mono<String>> getStatus(@Parameter(description = "Result UUID") @PathVariable("resultUuid") UUID resultUuid) {
+        Mono<String> result = service.getStatus(resultUuid);
+        return ResponseEntity.ok().body(result);
+    }
+
+    @PutMapping(value = "/results/{resultUuid}/invalidateStatus", produces = APPLICATION_JSON_VALUE)
```

Comment:
/results/{resultUuid}/invalidate-status ? snake case seems to be usual for REST API

Reply:
Changed